src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const allHeadlines = document.querySelectorAll('th');
+const tBody = document.querySelector('tbody');
+const allRows = Array.from(tBody.querySelectorAll('tr'));
+
+for (let i = 0; i < allHeadlines.length; i++) {
+  allHeadlines[i].addEventListener('click', () => {
+    const sortedRows = allRows.sort((a, b) => {
```

Comment:
The `sort` method is being called directly on the `allRows` array, which modifies the original array. This can lead to unexpected behavior if the table is sorted multiple times. Consider using `slice()` to create a shallow copy of the array before sorting to avoid modifying the original array.